src/test/kotlin/dk/cachet/carp/webservices/data/service/CoreDataStreamServiceTest.kt

```diff
@@ -0,0 +1,133 @@
+package dk.cachet.carp.webservices.data.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import dk.cachet.carp.common.application.UUID
+import dk.cachet.carp.common.infrastructure.serialization.COMMON_SERIAL_MODULE
+import dk.cachet.carp.common.infrastructure.serialization.createDefaultJSON
+import dk.cachet.carp.data.application.DataStreamBatch
+import dk.cachet.carp.data.application.MutableDataStreamBatch
+import dk.cachet.carp.data.infrastructure.DataStreamServiceRequest
+import dk.cachet.carp.webservices.data.domain.DataStreamSequence
+import dk.cachet.carp.webservices.data.repository.DataStreamConfigurationRepository
+import dk.cachet.carp.webservices.data.repository.DataStreamSequenceRepository
+import dk.cachet.carp.webservices.data.service.core.CoreDataStreamService
+import io.mockk.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.ArgumentMatchers.any
+import java.io.File
+import kotlin.test.assertFailsWith
+@OptIn(ExperimentalCoroutinesApi::class)
+class CoreDataStreamServiceTest {
+    private val dataStreamSequenceRepository: DataStreamSequenceRepository = mockk()
+    private val objectMapper: ObjectMapper = mockk()
+    private val json = createDefaultJSON(COMMON_SERIAL_MODULE)
+
+    companion object
+    {
+        private val dataStreamConfigurationRepository: DataStreamConfigurationRepository = mockk()
```

Comment:
Why is this in a companion object? Service declarations shouldn't be part of the companion object, this is like declaring something as static in java.

Reply:
+